spring-session-docs/modules/ROOT/nav.adoc

```diff
@@ -15,7 +15,6 @@
 *** xref:guides/boot-webflux-custom-cookie.adoc[Custom Cookie]
 ** Java Configuration
 ** XML Configuration
-* xref:modules.adoc[Modules]
```

Comment:
The documentation section describing modules arrangement was introduced in 2.0 when project was split into separate repos and the BOM was originally introduced. With everything now being in the same repo, I thought this section could safely be removed.